scripts/com.r2studio.TsumBeta/src/index.js

```diff
@@ -37,9 +37,11 @@ function log() {
   sleep(10);
   var args = [];
   if (ts !== undefined && ts.showHeartLog && ts.record && ts.record['hearts_count']) {
-    var msg = "";
-    msg += "R:"+ts.record['hearts_count'].receivedCount+" ";
-    msg += "S:"+ts.record['hearts_count'].sentCount;
+    var msg = "C";
+    // msg += "R:"+ts.record['hearts_count'].receivedCount+" ";
+    // msg += "S:"+ts.record['hearts_count'].sentCount;
+    // add timestamp in HH:MM:SS
+    msg += new Date().toTimeString().split(' ')[0];
```

Comment:
Yep, forgot to take this out, thanks. Will do that. 

I think it might be useful to have this as an option (like the debug logs) but that should be its own pull request if so.